src/components/Main/Main.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext } from 'react';
+import { TodoList } from '../TodoList/TodoList';
+import { DispatchContext, TodosContext } from '../../Store';
+import { Todo } from '../../Types/Todo';
+
+export const Main: React.FC = () => {
+  const {
+    todos,
+    todosCompleted,
+    selectedActive,
+    selectedCompleted,
+  } = useContext(TodosContext);
+  const dispatch = useContext(DispatchContext);
+
+  const handleChange = () => {
+    dispatch({ type: 'todosCompleted' });
```

Comment:
The dispatch action type 'todosCompleted' is not clear. It would be better to have a more descriptive action type such as 'toggleTodosCompleted'.